### ISSUE TITLE
URIs should be case-sensitive

### ISSUE THREAD

**Issue by warpech** [Starcounter/Home#433]
Right now, Starcounter handles URIs in a case-insensitive way. This leads to app authors requesting resources using any possible combination:

```
/sys/uniform.css/uniform.css
/sys/Uniform.css/Uniform.CSS
/sys/Uniform.css/Uniform.css
```

This appears to work from the app authors' perspective, but is wrong, because for a Web browser, the URLs are **case sensitive**. Browser cannot reuse a resource from cache if it is loaded from another resource.

In best case, this will lead to bad performance. In the worst case, this will lead to incorrect rendering of CSS stylesheets and JavaScript exceptions about disallowed re-registration of a custom element.

Expected: 

When the file path is `/sys/uniform.css/uniform.css`, requesting it as `/sys/Uniform.css/Uniform.css` should give a 404.

**Comment by miyconst** [comment]
This is due to Windows file system being case-insensitive. I am not sure if we shall do anything about it, since it will be resolved automatically when moving to Nova and Linux.

**Comment by warpech** [comment]
The problem also affects URI handled with `Handle.GET`, so it is not only Windows problem but also a problem with matching of URIs in our HTTP handlers.

**Comment by miyconst** [comment]
Closing as won't fix in Starcounter 2.4 and not applicable in Starcounter 3.0.
